### ISSUE TITLE
Potential bug in `oscarapi_post_checkout` signal being sent

### ISSUE THREAD

**Issue by yvandermeer** [django-oscar/django-oscar-api#136]
In `oscarapi/views/checkout.py`, the "response" argument of the `oscarapi_post_checkout` signal being sent is the entire `rest_framework.response`. I don't think that was the intent of the code.
```
from rest_framework import response


class CheckoutView(views.APIView):
    ...

    def post(self, request, format=None):
        ...
        oscarapi_post_checkout.send(
            sender=self, order=order, user=request.user,
            request=request, response=response)
        return response.Response(o_ser.data)
```
Probably should be something like this:
```
from rest_framework import response


class CheckoutView(views.APIView):
    ...

    def post(self, request, format=None):
        ...
        response = response.Response(o_ser.data)
        oscarapi_post_checkout.send(
            sender=self, order=order, user=request.user,
            request=request, response=response)
        return response
```

**Comment by specialunderwear** [comment]
syntax error after =

**Comment by yvandermeer** [comment]
Edited to fix syntax error. ;)

**Comment by maerteijn** [comment]
Nice! It's really handy to have the rest framework response module in the signal instead of the response object :) Going to fix it

**Comment by specialunderwear** [comment]
I think this is expres really

**Comment by maerteijn** [comment]
Fixed in 1d92efccb9de215fd7a0c856dac4d8e17a47a2b1, released 1.5.1
